src/mappings.js

```diff
@@ -69,7 +69,9 @@ export const fieldsToIncludeFields = fields => {
   const inputMappings = Object.keys(mappings);
 
   return fields
-    .map(field => {
+    .map(nestedField => {
```

Comment:
This field is not always nested. I would rename to something else. Perhaps `f`.

Reply:
ok!

Reply:
If it's still a `field`, why not leave it named `field`?

Reply:
I say this because I typically reserve the `f` variable name for certain functions, as this is a mathematical paradigm.